executor/aggfuncs/func_max_min_test.go

```diff
@@ -93,3 +94,56 @@ func (s *testSuite) TestMaxMin(c *C) {
 		s.testAggFunc(c, test)
 	}
 }
+
+func (s *testSuite) TestMemMaxMin(c *C) {
+	tests := []aggMemTest{
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeLonglong, 5,
+			aggfuncs.DefPartialResult4MaxMinIntSize, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeLonglong, 5,
+			aggfuncs.DefPartialResult4MaxMinUintSize, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeNewDecimal, 5,
+			aggfuncs.DefPartialResult4MaxMinDecimalSize, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeFloat, 5,
+			aggfuncs.DefPartialResult4MaxMinFloat32Size, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeDouble, 5,
+			aggfuncs.DefPartialResult4MaxMinFloat64Size, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeDate, 5,
+			aggfuncs.DefPartialResult4TimeSize, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeDuration, 5,
+			aggfuncs.DefPartialResult4MaxMinDurationSize, defaultUpdateMemDeltaGens, false),
+		buildAggMemTester(ast.AggFuncMax, mysql.TypeString, 5,
```

Comment:
try 5 -> 100?

Reply:
only `buildAggMemTester(ast.AggFuncMax, mysql.TypeSet...` not pass test, Do you know why?